### ISSUE TITLE
Need a wireframe?

### ISSUE THREAD

**Issue by vijayxtreme** [team-hack/react-components#1]
I think we should probably get a wireframe going for this project, so we can figure out what kinds of reusable components we can create.  Maybe something done in Figma, or equivalent?

**Comment by vijayxtreme** [comment]
Basic wireframe using Figma https://www.figma.com/file/MutJR1XVX6qcrwLIm7yswB/Components-UI-Wireframe?node-id=0%3A1

**Comment by airandfingers** [comment]
Hey @vijayxtreme I'm interested in anything we're doing in Figma, but not entirely clear on what this is for, since it sounded like developers can more or less dictate what the components will be.

At a minimum, I can mock up any components I'm going to build, before I build them.. that'd be good practice for me.

**Comment by vijayxtreme** [comment]
@airandfingers Right, I think let's just continue to build a wireframe in parallel in Figma.  I think if you want to build your own too and attach as a link to this "issue" that's cool, or we can collab on the one that exists.  The wireframe is to just help give people an idea of things to shoot for over time -- like maybe moving from making simple buttons and navbars to full featured landing pages.
